managed-cluster-config-services/manged-cluster-config.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: 00-osd-manged-cluster-config.selectorsyncset.yaml 
+  url: https://github.com/openshift/managed-cluster-config 
+  hash_length: 7
+  environments:
+  - name: production
+    parameters:
```

Comment:
@jmelis should this be blank or removed completely?

Reply:
you can actually remove the whole `environments` key